src/main/java/nl/esciencecenter/restape/ToolBenchmarkingAPIs.java

```diff
@@ -118,19 +135,27 @@ private static JSONObject computeBiotoolsBenchmark(SolutionWorkflow workflow) {
             "Number of tools which support Windows OS.", "operatingSystem", "Windows");
       benchmarks.put(BioToolsBenchmark.countOSEntries(biotoolsAnnotations, windowsBenchmark).getJson());
 
-      BenchmarkBase bioToolBenchmark = new BenchmarkBase("In bio.tools", "Available in bio.tools",
-            "Number of tools annotated in bio.tools.", null, null);
-      benchmarks.put(BioToolsBenchmark.countEntries(biotoolsAnnotations, bioToolBenchmark).getJson());
-
-      // BenchmarkBase openEBenchmark = new BenchmarkBase("In OpenEBench", "Available
-      // in OpenEBench",
-      // "Number of tools tracked in OpenEBench.", null, null);
-      // benchmarks.put(BioToolsBenchmark.countEntries(biotoolsAnnotations,
-      // bioToolBenchmark).getJson());
-
-      benchmarkResult.put("benchmarks", benchmarks);
-
-      return benchmarkResult;
+      /*
```

Comment:
I'm sure this is commented out with a reason, but maybe include the reason or a TODO with it?

Reply:
Good point, I might just remove the commented part.